README.md

```diff
@@ -69,110 +90,96 @@ pip install -i https://test.pypi.org/simple/ taichi-nightly
-<a href="https://github.com/taichi-dev/quantaichi"><img src="https://raw.githubusercontent.com/taichi-dev/public_files/master/taichi/fluid_with_bunnies.gif" height="213px"></a>
+- If you spot an technical or documentation issue, file an issue at [GitHub Issues](https://github.com/taichi-dev/taichi/issues)
+- If you spot any security issue, mail directly to <a href = "mailto:security@taichi.graphics?subject = Taichi Security Problem">security@taichi.graphics</a>.
 
-## Lectures & talks
-- **SIGGRAPH 2020 course on Taichi basics**: [YouTube](https://youtu.be/Y0-76n3aZFA), [Bilibili](https://www.bilibili.com/video/BV1kA411n7jk/), [slides (pdf)](https://yuanming.taichi.graphics/publication/2020-taichi-tutorial/taichi-tutorial.pdf).
-- Chinagraph 2020 用太极编写物理引擎: [哔哩哔哩](https://www.bilibili.com/video/BV1gA411j7H5)
-- GAMES 201 高级物理引擎实战指南2020: [课件](https://github.com/taichi-dev/games201)
-- 太极图形课第一季：[课件](https://github.com/taichiCourse01)
-- [TaichiCon](https://github.com/taichi-dev/taichicon): Taichi developer conferences
-- More to come...
+### Contact us
 
----
+You can also join our community from Slack or WeChat. Drop us a message at <a href = "mailto:contact@taichi.graphics">contact@taichi.graphics</a> first, and we'll follow up.
+
+### Follow us on social media
+
+- [Twitter](https://twitter.com/TaichiGraphics)
+- [LinkedIn](https://www.linkedin.com/company/taichi-graphics/)
+- [YouTube](https://www.youtube.com/channel/UCu-k1Wglo9Ll_o2j5Bxl4cw/featured)
+
+### Join us
 
-If you use Taichi in your research, please cite related papers:
+The Taichi programming language is powered by Taichi Graphics. The team is rapidly expanding. Find us from the following channels to get a taste of our *vibe*. If you feel you can fit in, welcome to join us!:-)
 
-- [**(SIGGRAPH Asia 2019) Taichi: High-Performance Computation on Sparse Data Structures**](https://yuanming.taichi.graphics/publication/2019-taichi/taichi-lang.pdf) [[Video]](https://youtu.be/wKw8LMF3Djo) [[BibTex]](https://raw.githubusercontent.com/taichi-dev/taichi/master/misc/taichi_bibtex.txt) [[Code]](https://github.com/taichi-dev/taichi)
-- [**(ICLR 2020) DiffTaichi: Differentiable Programming for Physical Simulation**](https://arxiv.org/abs/1910.00935) [[Video]](https://www.youtube.com/watch?v=Z1xvAZve9aE) [[BibTex]](https://raw.githubusercontent.com/taichi-dev/taichi/master/misc/difftaichi_bibtex.txt) [[Code]](https://github.com/yuanming-hu/difftaichi)
-- [**(SIGGRAPH 2021) QuanTaichi: A Compiler for Quantized Simulations**](https://yuanming.taichi.graphics/publication/2021-quantaichi/quantaichi.pdf) [[Video]](https://www.youtube.com/watch?v=0jdrAQOxJlY) [[BibTex]](https://raw.githubusercontent.com/taichi-dev/taichi/master/misc/quantaichi_bibtex.txt) [[Code]](https://github.com/taichi-dev/quantaichi)
+- [Taichi Graphics](https://taichi.graphics/)
+- [Careers at Taichi](https://taichi.graphics/careers/)
```

Comment:
I'm not entirely sure if we should put too much company info here. Maybe we should leave the open-source README more focused on the project itself?

Reply:
In the longer term, we should set up twitter\LinkedIn\YouTube accounts for the Taichi Language. I think, for now, we can put company information here, because Taichi Graphics is the driver of the Taichi programming language. For recruiting, we also hope more enterprise users post their job openings here. 
That's my reasoning, but will yield to your decision.

Reply:
Re: " I feel like the contents may duplicate what the new Taichi website would cover."  Some (or many) developers do not use Taichi's official website and use GitHub only. So it makes sense to me to echo the Taichi's official site. 
I have to agree that "develop elegantly" and "run rapidly" are more marketing style do not fit into the "geek" vibe in GitHub. I can replace them with "Fast development" and "High performance". Let me know if I have properly understood your questions. @yuanming-hu

Reply:
+1, I also prefer that the readme is mostly project focused.